### ISSUE TITLE
Why doesn't releasing a candidate also release the candidate's documentation?

### ISSUE THREAD

**Issue by expipiplus1** [haskell/hackage-server#879]
or: Why does the maintainer have to upload documentation twice but source only once?

To reproduce

- Upload a package candidate 
- Upload documentation
- Observe that the documentation is browsable
- Release candidate
- Observe that the documentation is not available

**Comment by expipiplus1** [comment]
Ah, looks like a duplicate of https://github.com/haskell/hackage-server/issues/70

**Comment by expipiplus1** [comment]
Thanks! Much more convenient!
